src/components/app/Home.js

```diff
@@ -384,45 +316,22 @@ class HomeImpl extends React.PureComponent<HomeProps, HomeState> {
                 profiler.firefox.com.
               </Localized>
             </p>
-          </div>
-          {/* end of grid container */}
-        </div>
-      </InstructionTransition>
-    );
-  }
-
-  _renderInstallAddonInstructions() {
-    return (
-      <InstructionTransition key={0}>
-        <div
-          className="homeInstructions"
-          data-testid="home-install-addon-instructions"
-        >
-          {/* Grid container: homeInstructions */}
-          {/* Left column: img */}
-          <img
-            className="homeSectionScreenshot"
-            src={PerfScreenshot}
-            alt="screenshot of profiler.firefox.com"
-          />
-          {/* Right column: instructions */}
-          <div>
-            <InstallButton
-              name="Gecko Profiler"
-              className="homeSectionButton"
-              xpiUrl={ADDON_URL}
-            >
-              <span className="homeSectionPlus">+</span>
-              <Localized id="Home--addon-button">Install add-on</Localized>
-            </InstallButton>
-            <DocsButton />
-            <p>
-              <Localized id="Home--addon-button-instructions">
-                Install the Gecko Profiler Add-on to start recording a
-                performance profile in Firefox, then analyze it and share it
-                with profiler.firefox.com.
+            {webChannelAvailable ? null : (
+              <Localized
+                id="Home--web-channel-unavailable"
+                elems={{
+                  code: <code />,
+                }}
+              >
+                <p data-testid="home-web-channel-unavailable-message">
+                  This profiler instance was unable to connect to the
+                  WebChannel. This usually means that it&apos;s running on a
+                  different host from the one that is specified in the
+                  preference{' '}
+                  <code>devtools.performance.recording.ui-base-url</code>.
+                </p>
```

Comment:
I feel like the text should be styled differently. This is an error but it doesn't look like so. I wonder if this should be completely in the disabled button's title instead of having it here, what do you think?

Reply:
Or maybe we should also replace this text that's above: 
```
                Enable the profiler menu button to start recording a performance
                profile in Firefox, then analyze it and share it with
                profiler.firefox.com.
```
and have something more consistent.

Reply:
> I feel like the text should be styled differently. This is an error but it doesn't look like so.

Well, it's legitimate state to be in. Our users will probably never see it. And profiler developers run the profiler from localhost:4224 all the time and would always see it, but in that case it's expected and not really an error.

> I wonder if this should be completely in the disabled button's title instead of having it here, what do you think?

That's an interesting alternative. But if it's only in the title, it is hard to select + copy the pref name. But maybe that's ok.

Reply:
> Or maybe we should also replace this text that's above:
> 
> ```
>                 Enable the profiler menu button to start recording a performance
>                 profile in Firefox, then analyze it and share it with
>                 profiler.firefox.com.
> ```
> 
> and have something more consistent.

I'm not exactly sure what you mean by that. Do you have a suggestion for a more consistent replacement?

Reply:
In the case of error, I suggest to replace:

> Enable the profiler menu button to start recording a performance profile in ⁨Firefox⁩, then analyze it and share it with profiler.firefox.com.
>
> This profiler instance was unable to connect to the WebChannel. This usually means that it's running on a different host from the one that is specified in the preference devtools.performance.recording.ui-base-url.

With

> This profiler instance was unable to connect to the WebChannel. This usually means that it's running on a different host from the one that is specified in the preference devtools.performance.recording.ui-base-url. Please check and change its value in about:config if necessary.

I believe that in the cases where this error happens, we don't need the `Enable the profiler menu...` text. What do you think?

I still would like a different styling but I don't have a good idea... Maybe adding an icon such as https://github.com/firefox-devtools/profiler/blob/main/res/img/svg/error-red.svg or https://github.com/firefox-devtools/profiler/blob/main/res/img/svg/warning.svg? What do you think?

Reply:
Thanks for these suggestions! I've incorporated them mostly. I still believe that a warning styling is not necessary. The message will only be seen by profiler developers, and it would be annoying to think "something's wrong!" every time I open a deploy preview, for example.

I've also changed the message to answer the question "Why should I care about this error" - specifically, that the pref allows profile capturing and programmatic control over the menu button from the profiler instance.